tox.ini

```diff
@@ -1,13 +1,23 @@
 # This file is part of REANA.
-# Copyright (C) 2022, 2023 CERN.
+# Copyright (C) 2022, 2023, 2024 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 [tox]
-envlist = py38, py39, py310, py311, py312
+envlist =
+    py38
+    py39
+    py310
+    py311
+    py312
 
 [testenv]
-deps = pytest
-       pytest-cov
-commands = pytest {posargs}
+deps =
+    pytest
+    pytest-cov
+commands =
+    pip install -e .[all]
```

Comment:
Shall we use `extras` instead of manually installing the packages in commands? https://tox.wiki/en/latest/config.html#extras